### ISSUE TITLE
CEF applications ran under proxychains only display white windows

### ISSUE THREAD

**Issue by kb-1000** [rofl0r/proxychains-ng#329]
This both happens with the official Minecraft launcher and Jetbrains Toolbox:
![image](https://user-images.githubusercontent.com/18211838/83844693-c03c2c00-a707-11ea-8467-be1307eb690c.png)
When I don't run them under proxychains, they work as expected, but obviously get no internet connection. This is probably not limited to Minecraft and Jetbrains Toolbox, however I only tested it with them.

**Comment by rofl0r** [comment]
these are java apps i suppose ?

**Comment by kb-1000** [comment]
No. JavaScript maybe, but not Java... and java has nothing to do with js.

**Comment by rofl0r** [comment]
well, as far as i'm aware minecraft is written in java, at least the original version. and the java loader does some low-level hacks that are incompatible with DSO preloading, proxychains never worked for these programs.

**Comment by kb-1000** [comment]
Minecraft itself, yes. But I'm talking about the Minecraft _launcher_.
